libr/core/cbin.c

```diff
@@ -817,7 +817,7 @@ static int bin_info(RCore *r, int mode) {
 			}
 		}
 		if (IS_MODE_JSON (mode)) {
-			r_cons_printf ("}");
+			r_cons_printf ("}\n");
```

Comment:
those newlines where there when printing multiple outputs at once so the json looks in one line. maybe this newline thing should go somewhere else